t/18-backend-qemu.t

```diff
@@ -87,6 +87,17 @@ subtest 'eject cd' => sub {
     is_deeply $called{handle_qmp_command}[1], \%custom_remove_params, 'blockdev-remove-medium called with custom parameters';
 };
 
+subtest 'switch_network' => sub {
+    my %switch_network_params = ('arguments' => {'name' => 'qanet0', 'up' => Mojo::JSON->false}, 'execute' => 'set_link');
+    $called{handle_qmp_command} = undef;
+
+    $backend_mock->redefine(handle_qmp_command => sub { push @{$called{handle_qmp_command}}, $_[1] });
+    $backend->switch_network({network_state => 0});
+    ok(exists $called{handle_qmp_command}, 'a qmp command has been called');
+    is_deeply($called{handle_qmp_command}[0], \%switch_network_params, 'qmp command for setlink is passed') or diag explain $called{handle_qmp_command};
+    $called{handle_qmp_command} = undef;
+};
+
```

Comment:
something from here is causing unhandled test output with a bare "1" that should be prevented or caught with Test::Output, see https://github.com/os-autoinst/os-autoinst/runs/5156844864?check_suite_focus=true#step:3:589

Reply:
Yeah, was actually a `bmwqemu::diag` call on the `backend::qemu::switch_network` implementation